dependency/engine.go

```diff
@@ -669,9 +674,6 @@ func (engine *Engine) gotStopped(name string, err error, resourceLog []resourceA
 
 			// Something went wrong but we don't know what. Try again soon.
 			logFn("%q manifold worker returned unexpected error: %v", name, err)
-			if tracer, ok := err.(stackTracer); ok {
```

Comment:
Did we still want this here as well? Or will a restart first stop and the start such that we still get the stack trace added above?